arb_os/accounts.mini

```diff
@@ -362,6 +363,28 @@ public func account_setBaseTxFeeL1Gas(account: Account, fee: uint) -> Account {
     };
 }
 
+public view write func account_setStorage(account: Account, storage: StorageMap) -> Account {
+    return account with {
+        contractInfo: Some(
+            xif let Some(contractInfo) = account.contractInfo {
+                contractInfo with {
+                    storage: storage
+                }
+            } else {
+                let fakeCode = bytearray_new(0);
+                fakeCode = bytearray_setByte(fakeCode, 0, 0xfe);
+                struct {
+                    contractStateVersion: 1,
```

Comment:
I would use `contractStateVersion: const:: ContractStateVersion_current,` here because that is future-proof if we change the contract state version.